cmd/telemetrygen/internal/metrics/worker.go

```diff
@@ -5,6 +5,7 @@ package metrics
 
 import (
 	"context"
+	rand "math/rand/v2"
```

Comment:
All our other metrics are deterministic, I think the histogram one should be deterministic as well

Reply:
Thanks for the comment. These metrics are still deterministic, since the PRG uses the [same seed](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/7f167a274a2e548acc8f3871a6d1cbd0a3b83622/cmd/telemetrygen/internal/metrics/worker.go#L42). So the generated histograms will be same across multiple runs.

Reply:
Ah, still, I would prefer to not do the calculation but instead have the actual numbers from the histogram hardcoded. Could you do this?

Reply:
Removed the generator. 
Now the histogram is generated based on the iteration number.